### DISCUSSION TITLE
Ödev Teslimi

### DISCUSSION THREAD

**Discussion by haticetekeli** [gusanmaz/bmb212-algorithms-2024#6] [General]
Youtube linkini oluşturduğumuz txt dosyasının içine mi koyalım? ve her bir başlık için bir video istemişsiniz bunu bir oynatma listesi oluşturup videoları bu listenin altına eklesek olur mu? Kendi adımıza olmayan bir youtube hesabı kullansak sıkıntı olur mu? İlk başta çözdüğümüz konu anlatımlı problemleri de ekleyecek miyiz?

**Comment by gusanmaz** [comment]
> Youtube linkini oluşturduğumuz txt dosyasının içine mi koyalım?

Evet

> her bir başlık için bir video istemişsiniz bunu bir oynatma listesi oluşturup videoları bu listenin altına eklesek olur mu?

Olur

> Kendi adımıza olmayan bir youtube hesabı kullansak sıkıntı olur mu?

Sorun olmaz

> İlk başta çözdüğümüz konu anlatımlı problemleri de ekleyecek miyiz?

Kodlar eklenecek ama bu problemlerden cekeceginiz videoda bahsetmeniz istenmiyor.
